packages/shortest/README.md

```diff
@@ -136,4 +136,17 @@ GITHUB_TOTP_SECRET=your_secret  # Only for GitHub auth tests
 You can run shortest in your CI/CD pipeline by running tests in headless mode. Make sure to add your Anthropic API key to your CI/CD pipeline secrets.
 
 ## Documentation
-Visit [GitHub](https://github.com/anti-work/shortest) for detailed docs
\ No newline at end of file
+Visit [GitHub](https://github.com/anti-work/shortest) for detailed docs
+
+### Prerequisites
+
+- Node.js
+- pnpm
```

Comment:
I don't think pnpm should be a prerequisite as many/most projects don't use it.

Reply:
That's right. Delete that part in the new commit.